app/src/main/java/com/poupa/vinylmusicplayer/ui/activities/AlbumDetailActivity.java

```diff
@@ -116,6 +118,8 @@
     private MaterialDialog wikiDialog;
     private LastFMRestClient lastFMRestClient;
 
+    private final WeakMethodReference<AlbumDetailActivity> onDiscographyChanged = new WeakMethodReference<>(this, AlbumDetailActivity::reload);
```

Comment:
Is there any chance to move this ref and the onStart/Stop/Restart implementation to the base class AbsSlidingMusicPanelActivity, i.e. avoid copy/paste?

Reply:
@Iislsdum any update on this please?

Reply:
Resolved in 48476d1